packages/contracts/contracts/DependencyRegistryV0.sol

```diff
@@ -633,11 +669,49 @@ contract DependencyRegistryV0 is
         return _dependencyTypes.at(_index).toString();
     }
 
+    /**
+     * @notice Returns number of registered license types
+     * @return Number of registered license types.
+     * @dev This is only intended to be called outside of block
+     * execution where there is no gas limit.
+     */
+    function getLicenseTypes() external view returns (string[] memory) {
+        string[] memory licenseTypes = new string[](
+            _licenseTypes.length()
+        );
+        uint256 numLicenseTypes = _licenseTypes.length();
```

Comment:
nit - would rather switch the order of these two statements, and only call `_licenseTypes.length()` once. We should follow that pattern in some of the other view functions in this contract as well (might make sense as follow-on work if you would like!)